routes/api.php

```diff
@@ -2,129 +2,134 @@
 
 use Illuminate\Support\Facades\Route;
 
-Route::apiResource('statistics', 'Statistics\\ApiStatisticsController', ['only' => ['index']]);
+Route::apiResource('statistics', 'Statistics\\ApiStatisticsController', ['only' => ['index']])->name('index', 'api.statistics');
 
 Route::resource('compliance', 'Settings\\ApiComplianceController', ['only' => ['index', 'show']]);
 
-Route::resource('currencies', 'Settings\\ApiCurrencyController', ['only' => ['index', 'show']]);
+Route::resource('currencies', 'Settings\\ApiCurrencyController', ['only' => ['index', 'show']])->name('index', 'api.currencies');
 
-Route::group(['middleware' => ['auth:api']], function () {
+Route::group(['middleware' => ['auth:apiext']], function () {
     Route::get('/', 'ApiController@success')->name('api');
+    Route::name('api.')->group(function () {
+        // Me
+        Route::get('/me', 'Account\\ApiUserController@show');
+        Route::get('/me/compliance', 'Account\\ApiUserController@compliance');
+        Route::get('/me/compliance/{id}', 'Account\\ApiUserController@get');
+        Route::post('/me/compliance', 'Account\\ApiUserController@set');
 
-    // Me
-    Route::get('/me', 'Account\\ApiUserController@show');
-    Route::get('/me/compliance', 'Account\\ApiUserController@compliance');
-    Route::get('/me/compliance/{id}', 'Account\\ApiUserController@get');
-    Route::post('/me/compliance', 'Account\\ApiUserController@set');
+        // Contacts
+        Route::apiResource('contacts', 'ApiContactController')
+      ->names(['index' => 'contacts', 'show' => 'contact']);
+        Route::put('/contacts/{contact}/setMe', 'ApiContactController@setMe');
+
+        // Genders
+        Route::apiResource('genders', 'Account\\ApiGenderController');
+
+        // Relationships
+        Route::apiResource('relationships', 'ApiRelationshipController', ['except' => ['index']])
+      ->name('show', 'relationship');
```

Comment:
Indentation?

Reply:
This is due to StyleCI which has broken my indentation! I'll fix it